target/scripts/build/packages.sh

```diff
@@ -111,7 +146,7 @@ function _install_dovecot() {
     _log 'trace' 'Using Dovecot community repository'
     curl https://repo.dovecot.org/DOVECOT-REPO-GPG | gpg --import
     gpg --export ED409DA1 > /etc/apt/trusted.gpg.d/dovecot.gpg
-    echo "deb https://repo.dovecot.org/ce-2.3-latest/debian/bullseye bullseye main" > /etc/apt/sources.list.d/dovecot.list
+    echo "deb https://repo.dovecot.org/ce-2.3-latest/debian/${VERSION_CODENAME} ${VERSION_CODENAME} main" > /etc/apt/sources.list.d/dovecot.list
```

Comment:
```suggestion
    # VERSION_CODENAME sourced from /etc/os-release
    echo "deb https://repo.dovecot.org/ce-2.3-latest/debian/${VERSION_CODENAME} ${VERSION_CODENAME} main" > /etc/apt/sources.list.d/dovecot.list
```

Reply:
PS: The same goes for here: https://github.com/docker-mailserver/docker-mailserver/pull/3403/files#diff-95c6b88025d3f0c1a18f85c2c05a14faa73ff59b57f6e938df5113221c8018f1R36